Automate/RedHatConsulting_Satellite6/Integration/Satellite/StdLib.class/__methods__/satellitecore.rb

```diff
@@ -0,0 +1,95 @@
+#
+# Description: Collection of Core Methods to use when interacting with Red Hat Satellite 6
+#
+
+module RedHatConsulting_Satellite6
+  module StdLib
+    module SatelliteCore
+
+      require 'apipie-bindings'
+      include RedHatConsulting_Utilities::StdLib::Core
+
+      SATELLITE_CONFIG_URI           = 'Integration/Satellite/Configuration/default'
+
+      def initialize( handle = $evm)
+        @handle                  = handle
```

Comment:
Seems like a lot of spaces betweeen variable name and =

Reply:
Good catch - will update